examples/assistants-azure.ts

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env -S npm run tsn -T
+
+import { AzureOpenAI } from 'openai';
+
+/**
+ * Example of polling for a complete response from an assistant
+ */
+const apiVersion = '2024-02-15-preview';
```

Comment:
Should we move to a recent API version?
```suggestion
const apiVersion = '2024-05-01-preview';
```